modules/xfem/test/tests/moving_interface/tests

```diff
@@ -63,6 +63,16 @@
     design = 'LevelSetBiMaterialRankTwo.md LevelSetBiMaterialRankFour.md XFEMSingleVariableConstraint.md'
     issues = '#11259'
   [../]
+  [./moving_bimaterials_displaced_mesh]
```

Comment:
I just have two concerns:
1) This new test is essentially the same as `moving_bimaterials`. I would say to just modify `moving_bimaterials` instead.
2) I know it was pretty fast, but it needs to run in less than a second, and I think it was a little slower than that.

Reply:
Maybe you can make it run faster by starting the interface closer to the problem spot and just taking a couple of steps.

Reply:
As you suggested, I modified the original `moving_bimaterials` test. I got it to fail at the same mesh density at timestep 2 (instead of 6) with the test passing at timestep 3 (with the fix). I think this is about as short as I can make the test since the mesh needs to be displaced when it is close to the element boundary. Let me know if it works for you.